packages/pigeon/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 20.0.0
+
+* Moves all codec logic to single custom codec per file.
+* **Breaking Change** Limits the number of total custom types to 126.
```

Comment:
Maybe mention that if clients hit this limit, the workaround for now would be to split their interface definition into multiple files.